src/driver/mysql/MysqlQueryRunner.ts

```diff
@@ -3366,9 +3366,13 @@ export class MysqlQueryRunner extends BaseQueryRunner implements QueryRunner {
         return c
     }
 
-    protected async getVersion(): Promise<string> {
+    async getVersion(): Promise<string> {
         const result = await this.query(`SELECT VERSION() AS \`version\``)
```

Comment:
Would be nice to add a typing here, like `PostgresQueryRunner` has:

```suggestion
        const result: [{ version: string }] = await this.query(`SELECT VERSION() AS \`version\``)
```